integration/src/test/resources/org/mule/test/integration/notifications/pipeline-message-notification-test-flow.xml

```diff
@@ -37,16 +35,32 @@
     </flow>
 
     <flow name="service-4">
-        <test-components:queue-push config-ref="ow-out" />
+        <test-components:queue-push config-ref="ow-out"/>
     </flow>
 
     <flow name="service-5">
         <set-payload value="#[!null.a]"/>
         <error-handler>
             <on-error-continue>
-                <test-components:queue-push config-ref="owException-out" />
+                <test-components:queue-push config-ref="owException-out"/>
             </on-error-continue>
         </error-handler>
     </flow>
 
+    <flow name="nestedFlowFailingRoot">
+        <flow-ref name="nestedFlowFailingOne"/>
+    </flow>
+
+    <flow name="nestedFlowFailingOne">
+        <flow-ref name="nestedFlowFailingTwo"/>
+    </flow>
+
+    <flow name="nestedFlowFailingTwo">
+        <flow-ref name="nestedFlowFailingThree"/>
+    </flow>
+
+    <flow name="nestedFlowFailingThree">
+        <set-payload value="#[!null.a]"/>
```

Comment:
use a `raise-error` component unless you actually need the error to be an EXPRESSION error.